sdk/e2e/11_deploy_router_upi_traffic_split_test.py

```diff
@@ -175,7 +177,7 @@ def test_deploy_router_upi_traffic_split():
             variable_pb2.Variable(
                 name="client_id",
                 type=type_pb2.TYPE_STRING,
-                string_value="1234",
+                string_value="12",
```

Comment:
both 7 and 12 returns treatment. Hence how traffic rules is tested, is that two request both should get treatment, but the one not fulfilling traffic rule will get control, which is `12`.  `1234` previously returns treatment which calls control route